MetaGenerator/TimeKeeper.cs

```diff
@@ -59,18 +59,14 @@ private void PrintMeasurements()
         }
 
         private long TotalTime()
-        {
-            return _measurements.Select(m => m.ElapsedMilliseconds).Sum();
-        }
+            => _measurements
+            .Select(m => m.ElapsedMilliseconds).Sum();
```

Comment:
*indentation

Reply:
Addressed in 9fc64c88dcd906d2d0de07d7ad6281413165b994